contracts/modules/capital/Pool.sol

```diff
@@ -389,7 +448,9 @@ contract Pool is IPool, MasterAwareV2, ReentrancyGuard {
     require(ok, "Pool: Transfer failed");
 
     // Transfer cover assets. Start from 1 (0 is ETH)
-    uint coverAssetsCount = coverAssets.length;
+    uint coverAssetsCount = coverAssets.length;  // substract 1 as ETH is skiped
```

Comment:
typo skiped -> skipped

Also you are starting with an index of 1 below so it seems to me you shouldn't substract for ETH to be skipped. and you actually don't so the comment is misleading

Reply:
Oh nice catch! Yup that comment didn't get deleted from the old code. Will fix that today.